client/boot/index.js

```diff
@@ -239,6 +239,11 @@ function reduxStoreReady( reduxStore ) {
 				user_id: user.get().ID,
 				calypso_env: config( 'env_id' )
 			} );
+
+			if ( window && window.app && window.app.commitChecksum ) {
```

Comment:
FWIW. Even though it's unlikely that this file will ever be run on the server, the `window` check will error if it is. Use `typeof( window ) !== 'undefined'`, or check for `global.window` instead.